src/components/ToggleSwitch/index.tsx

```diff
@@ -53,7 +54,7 @@ const styles = StyleSheet.create({
     lineHeight: dimen.normalizeDimen(18),
     textAlign: 'center',
     textAlignVertical: 'center',
-    color: '#f4f3f4',
+    color: COLORS.whiteSmoke,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the previous one, replacing hard-coded color values with dynamic ones from a theme object.
```diff
-    color: '#f4f3f4',
+    color: COLORS.whiteSmoke,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->